web/src/app/administration/components/workbasket-access-items/workbasket-access-items.component.spec.ts

```diff
@@ -115,15 +115,16 @@ describe('WorkbasketAccessItemsComponent', () => {
     expect(debugElement.querySelectorAll('#table-access-items > tbody > tr').length).toBe(1);
   });
 
-  it('should show alert successfull after saving', async(() => {
+  it('should show success alert after saving', async(() => {
```

Comment:
we are not calling it alert anymore ;)

Reply:
The renaming is something i wanted to do in combination with the ranaming and splitting of the `NOTIFICATION_TYPES`.
If you think this is something, that should be done earlier i could do this now.